tests/publiccloud/check_registercloudguest.pm

```diff
@@ -148,6 +147,49 @@ sub check_instance_unregistered {
     }
 }
 
+sub new_registration {
+    my ($instance) = @_;
+    record_info('Starting registration...');
+    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
+    check_instance_registered($instance);
+    return 0;
+}
+
+sub test_container_runtimes {
+    my ($instance) = @_;
+    my $image = "registry.suse.com/bci/bci-base:latest";
+
+    record_info('Install docker');
+    $instance->ssh_script_output("sudo rm /root/.docker/config.json");    # workaround for https://bugzilla.suse.com/show_bug.cgi?id=1231185
+    $instance->ssh_script_output("sudo zypper install -y docker");
+    record_info('Test docker');
+    $instance->ssh_script_output("sudo systemctl start docker.service");
+    record_info("systemctl status docker.service", script_output("systemctl status docker.service"));
```

Comment:
Here you are testing the docker service from the jump host and not the SUT.  To fix:

```suggestion
    record_info("systemctl status docker.service", ssh_script_output("systemctl status docker.service"));
```

Reply:
Good catch!